browser-test/src/applicant/application_version_fast_forward.test.ts

```diff
@@ -984,6 +984,407 @@ test.describe('Application Version Fast-Forward Flow', () => {
+        await expect(
+          applicantActor.getCardListLocator(
+            ApplicationStatusCardGroupName.NotStarted,
+          ),
+        ).not.toBeAttached()
+        await expect(
+          applicantActor.getCardListLocator(
+            ApplicationStatusCardGroupName.InProgress,
+          ),
+        ).toBeAttached()
+
+        const headingLocator = applicantActor.getCardHeadingLocator(
+          ApplicationStatusCardGroupName.InProgress,
+        )
+        await expect(headingLocator).toBeAttached()
+        expect(
+          await applicantActor.getProgramIdFromLocator(headingLocator),
+        ).toBe(programIdV2)
+      })
+
+      await test.step('As applicant - navigate to a question edit page and wait there', async () => {
+        await applicantActor.clickApplyProgramButton()
+
+        expect(applicantActor.getProgramIdFromApplicationReviewUrl()).toBe(
+          programIdV2,
+        )
+        await applicantActor.reanswerQuestion(Question.G)
+        await applicantActor.reanswerQuestion(Question.F)
+      })
+    })
+
+    // Civiform Admin creates program v3
+    const programIdV3 =
+      await test.step('As civiform admin - active program v3', async () => {
+        await test.step('create new questions', async () => {
+          await civiformAdminActor.addQuestions([Question.H])
+        })
+
+        await test.step('edit program', async () => {
+          await civiformAdminActor.editProgram()
+        })
+
+        await test.step('remove questions from program', async () => {
+          await civiformAdminActor.removeQuestionsFromBlock([
+            {
+              block: Block.Fourth,
+              questions: [Question.E],
+            },
+          ])
+        })
+
+        return await test.step('add questions to program and republish program', async () => {
+          await civiformAdminActor.addQuestionsToExistingBlocks([
+            {
+              block: Block.Fourth,
+              questions: [Question.H],
+              eligibilityValue: `${Question.H}-text-answer`,
+            },
+          ])
+
+          const programIdV3 =
+            civiformAdminActor.getProgramIdFromEditProgramUrl()
+
+          await civiformAdminActor.disableProgram()
+          await civiformAdminActor.publish()
+          return programIdV3
+        })
+      })
+
+    // Verify we have a new program version id
+    expect(
+      programIdV2,
+      'Verify Program v2 and Program v3 have the different IDs.',
+    ).not.toEqual(programIdV3)
+    expect(
+      programIdV2,
+      'Verify Program v2 ID is less than Program v3 ID.',
+    ).toBeLessThan(programIdV3)
+
+    // Applicant submits application for program v1; it does not change to program v3
+    await test.step('As applicant - active program v3', async () => {
+      await applicantActor.getPage().reload()
+      await applicantActor
+        .getPage()
+        .waitForURL('**/programs/program-fastforward-example/disabled')
+    })
```

Comment:
**suggestion:** Looks like its missing the last piece here

Reply:
Oops, I moved it into the class, but forgot to swap it over.